ecpy_hqc_legacy/manifest.enaml

```diff
@@ -223,7 +222,10 @@ enamldef HqcLegacyManifest(PluginManifest):
                         driver = 'anritsu_signal_source:AnritsuMG3694'
                         model = 'MG3694'
                         kind = 'RF source'
-                        connections = {'VisaGPIB': {'resource_class': 'INSTR'}
+                        connections = {'VisaGPIB': {'resource_class': 'INSTR'},
+                                       'VisaTCPIP': {'resource_class': 'INSTR',
```

Comment:
Does this instrument supports TCPIP ? why do you use an adapter then ? If it does not please delete.

Reply:
It is an option of the instrument, so we should leave it.